plow_linter/tests/lint_has_registry_keyword.rs

```diff
@@ -0,0 +1,76 @@
+use harriet::TurtleDocument;
+use plow_linter::lint::Lint;
+use plow_linter::lints::HasRegistryKeyword;
+
+const REGISTRY_KEYWORD_BASE: &str = concat!(
+    include_str!("data/default_ttl_header"),
+    r#"
+registry:ontologyFormatVersion "v1" ;
+registry:canonicalPrefix "test" ;
+registry:packageVersion "=2.3.4" ;
+registry:packageName "@field33/valid" ;
+"#
+);
+
+// Checks that registry:keyword field's correct format is 50 chars long and email is validated.
```

Comment:
"and email is validated."

I don't think that's part of the keyword 🙃

Reply:
Ah you caught a copy paste 😅

Reply:
[01f7d54](https://github.com/field33/plow/pull/4/commits/01f7d54559e647c0eafc0b3b145e170459502f34)